p4a/pythonforandroid/bootstrap.py

```diff
@@ -249,16 +268,22 @@ def strip_libraries(self, arch):
             info('Python was loaded from CrystaX, skipping strip')
             return
         env = arch.get_env()
-        strip = which('arm-linux-androideabi-strip', env['PATH'])
-        if strip is None:
-            warning('Can\'t find strip in PATH...')
-            return
-        strip = sh.Command(strip)
-        filens = shprint(sh.find, join(self.dist_dir, 'private'),
-                         join(self.dist_dir, 'libs'),
+        tokens = shlex.split(env['STRIP'])
+        strip = sh.Command(tokens[0])
+        if len(tokens) > 1:
+            strip = strip.bake(tokens[1:])
+
+        libs_dir = join(self.dist_dir, '_python_bundle',
+                        '_python_bundle', 'modules')
+        if self.ctx.python_recipe.name == 'python2legacy':
```

Comment:
Why are we encountering this recipe?

Reply:
It looks like a check done by default in python-for-android (https://github.com/kivy/python-for-android).

> I would suggest exploring alternative ways to handle kivy dependencies without polluting the repo

The plan in the future is to move the react native app folder into a submodule, so the React Native and Python build / Java native code can be handled separately.